_unary_ufuncs.py

```diff
@@ -4,9 +4,10 @@
 import torch
 
 import _util
+from _ndarray import asarray_replacer_1
 
 
-
+@asarray_replacer_1
 def absolute(x, /, out=None, *, where=True, casting='same_kind', order='K',
```

Comment:
Note that in PyTorch we still support Python 3.7, so perhaps it'd be better not to use positional only args? cc @ezyang.

Reply:
Note that we want to be NumPy compatible. As such, implementing a superset of NumPy's functionality would also do. Dropping positional-only args would achieve this. But again, let's wait to see what Ed thinks, perhaps PyTorch is planning to drop Python 3.7 support, idk.

Reply:
> This is then a question of what numpy version we want to track here.

1.24 is coming out in 1-2 weeks, so probably that. Won't be much different than 1.23.x though. Although tracking evolving function signatures is always a pain, and we'll need some way of handling that at some point.

Reply:
we should be dropping 3.7 soon

Reply:
ok clarification: we will drop it when colab drops it lol